todo-app/src/TodoStatus.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+export default class TodoStatus extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      done: this.props.done || false,
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      done: !this.state.done
+    })
+  }
+
+
+  render() {
+    let doneState = 'Not Done'
+    if (this.state.done) {
```

Comment:
I like this rendering logic here, but you could have extracted it to a method, for example:

```js
renderDoneLabel(done) {
  return done ? 'Done' : 'Not Done';
}
```

and call it in the `label`.